<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Btn_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>LED ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>LED OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_exit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
